data/remnant missions.txt

```diff
@@ -1713,3 +1717,151 @@ mission "Remnant: Broken Jump Drive job"
+			`	"Captain <first>! What a pleasant surprise," he trills. "If you are looking for more work, I don't have anything for you right now."`
+			`	You tell Plume about how you visited Nenia and had a threatening encounter with a void sprite, and offer him the recording and container with the tentacle. Plume recoils from you in disgust. "After all our work rebuilding trust with the Archon and the void sprites, you go and risk it for... what, exactly? Sightseeing? Thrill of the hunt?" His chant settles into an angry tone. "Leave the recording on the console. I want no part of that tentacle, though."`
+			`	As you are leaving Plume's lab a technician pulls you aside. "Our records mention that the tentacles had some interesting properties that were useful in combat. If you are interested, meet me in the spaceport in a few hours."`
+
+
+
+mission "Remnant: Face to Maw 2"
+	name "Take the slime sample to Viminal"
+	description "The void sprite slime has properties of interest to a team of engineers on <planet>."
+	to offer
+		has "Remnant: Face to Maw 1: done"
+	source "Aventine"
+	destination "Viminal"
+	cargo "slime sample" 1
+	on offer
+		conversation
+			`When Plume finally arrives you can tell from his step that he is excited. He slides onto the bench opposite you and pulls out a datapad. "I admire your bravery, Captain. I don't think I would have the courage to hold my ground against a void sprite like that," he chants respectfully, then changes tone to one of wonder. "In that last moment it almost looked like it was sizing you up or examining you." He shakes his head with an amused expression. "I spend so much time studying non-human creatures I sometimes slip and start anthropomorphizing them." His voice takes on a more militant cadence. "That was probably some kind of territorial display, or possibly some kind of dominance challenge. We don't know enough about void sprite behaviors to speculate further, though.`
+			`	"Be that as it may be, the more immediate reward for your curiosity is in this sample," he trills with excitement. "Several of our labs on Viminal have been working on the development of a new generation of ship technology, and this substance contains some chemicals that could be a key component. I have already transmitted the data to them, but they would like to compare with the original. Could you take this sample to them?"`
+			choice
+				`	"Yes, I can."`
+					accept
+				`	"I would rather not."`
+			`	Plume stoically accepts that with a nod. "That is your choice. I will arrange for another ship to transport it."`
+			decline
+	on accept
+		event "remnant: penguin"
+	on complete
+		log "People" "Taely" `One of Taely's main priorities is improving the maneuverability characteristics of Remnant Ships. To this end she has been working on building a new prototype ship based on the void sprites.`
+		"remnant taely" ++
+		payment 1000000
+		conversation
+			branch taely
```

Comment:
There is WAY too much writing here. I have to scroll to be able to read all of it, which isn't something that should be happening.
![image](https://user-images.githubusercontent.com/17688683/65838752-75def180-e2d4-11e9-8552-5b60575f424c.png)
![image](https://user-images.githubusercontent.com/17688683/65838762-7ecfc300-e2d4-11e9-96e0-345ac50c63a6.png)
Either reduce how much is written, or add a choice or two to break it up.

Reply:
Yeah, that is rather long.

I'm not too sure how many choices I can add or how to reduce the amount written without cutting into the information provided.

That being said, I have added two breaks: 
Firstly, giving the player a choice when Taely asks them to follow her, the player can choose to catch up in a few minutes, and thus skip the three paragraphs of conversation before the ship reveal.
Secondly, giving the player a chance to respond in the middle of the converstion. Currently the two choices are: 
"So my data is helping to develop new technology?"
"That's good. Any chance of a bonus for me?"

Neither of them goes anywhere, it is just an opportunity to respond. If you have better ideas, I'm open for it.